TestComplex.c

```diff
@@ -0,0 +1,144 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Ayisha Sameera,GET.
+// ------------------------------------------------------------------
+// Test.c
+// Program on branch Test1.2
+// Program displays test cases of Op_Add() Op_Mul() and Op_Sub() from Complex.c
+// Program gets complex number(s) as input from user and displays the result.
+// ------------------------------------------------------------------------------------------------
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#include "Complex.h"
+
+typedef struct {
+   ComplexNumber num1;
+   ComplexNumber num2;
+   ComplexNumber out;
+}Test;
+
+static bool IsEqual (ComplexNumber num1, ComplexNumber num2) {
+   // Check if difference between the two floating point numbers is negligibly small
+   return ((num1.Real - num2.Real < 0.0001) && (num1.Img - num2.Img < 0.0001));
+}
+
+static PrintNum (ComplexNumber num, char* str) {
+   printf ("  %5.3f + i %5.3f%s", num.Real, num.Img, str);
+}
+
+static void Test_Op_Mul () {
+   Test input[] = { {{50,30},{60,70},{900,5300}},
+                    {{89,90},{89,-9},{8731,7209}},

```

Comment:
These tests can be automated. The values of expected output can be computed and compared.